go.mod

```diff
@@ -30,6 +30,10 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 	k8s.io/api v0.17.12
 	k8s.io/apimachinery v0.17.12
+	k8s.io/utils v0.0.0-20200327001022-6496210b90e8
 )
 
-replace k8s.io/client-go => k8s.io/client-go v0.17.12
+replace (
+	github.com/fi-ts/cloud-go => ../cloud-go
```

Comment:
Please push your changes to the cloud-go project and use the dependency from the commit to the repository. Otherwise we can't build this.